tests/framework/extensions/users/usermap.go

```diff
@@ -0,0 +1,21 @@
+package users
+
+import (
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+)
+
```

Comment:
Can we have a line explaning what this function is for?

Reply:
good catch. updated

Reply:
Thanks, Can we change this `UserConfig creates and returns a template or config to create users`  to `UserConfig sets and returns username and password of the user'?

Also can we move this to `user.go` instead ?